File: Views/src/app/_components/administrative/client/dto/client-dto.ts
import { ContactDto } from 'src/app/_shared/dtos/contact-dto';
import { AddressDto } from 'src/app/_shared/dtos/address-dto';
import { NetworkDevicesDto } from '../technician/infra/dto/network-devices-dto';
import { ServiceBudgetDto } from '../../local/services/service-budget/dto/service-budget-dto';
import { SourceCollectDeliverDto } from '../../local/out-sourced/dto/source-collect-deliver-dto';
import { DestinyCollectDeliverDto } from '../../local/out-sourced/dto/destiny-collect-deliver-dto';

export class ClientDto {
  id: number;
  name: string;
  cnpj: string;
  responsible: string;
  comments: string;
  assured: boolean;
  clienttype: string;
  payment: number;
  expiration: Date;
  disabled: boolean;
  toBusinessBox: boolean;
  discount: number;
  addressid: number;
  address: AddressDto;
  contactid: number;
  contact: ContactDto;
  netWorkDevices: NetworkDevicesDto[];
  ServicesBudgets: ServiceBudgetDto[];
  DestinyCollectsDelivers: DestinyCollectDeliverDto[];
  SourceCollectsDelivers: SourceCollectDeliverDto[];
}

File: Services/Dto/ClientDto.cs
using System;
using System.Collections.Generic;

namespace Services.Dto
{
    public class ClientDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string CNPJ { get; set; }
        public string Responsible { get; set; }
        public string Comments { get; set; }
        public bool Assured { get; set; }
        public string ClientType { get; set; }
        public decimal Payment { get; set; }
        public DateTime Expiration { get; set; }
        public bool Disabled { get; set; }
        public bool ToBusinessBox { get; set; }

        public decimal Discount { get; set; }
        public int AddressId { get; set; }
        public AddressDto Address { get; set; }
        public int ContactId { get; set; }
        public ContactDto Contact { get; set; }
        public List<NetworkDevicesDto> NetWorkDevices { get; set; }
        public List<ServiceBudgetDto> ServicesBudgets { get; set; }
        public List<DestinyCollectDeliverDto> DestinyCollectDelivers { get; set; }
        public List<SourceCollectDeliverDto> SourceCollectDelivers { get; set; }
        public string ToSeach { get; set; }
    }
}